L05-myJUnit/src/main/java/ru/otus/chernovsa/myjunit/TestRunner.java

```diff
@@ -0,0 +1,89 @@
+package ru.otus.chernovsa.myjunit;
+
+import ru.otus.chernovsa.myjunit.annotation.After;
+import ru.otus.chernovsa.myjunit.annotation.Before;
+import ru.otus.chernovsa.myjunit.annotation.Test;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.*;
+
+public class TestRunner<T> {
+    private Class<T> clazz;
+    private List<T> listInstances = new ArrayList<>();
+    private List<String> testMethods = new ArrayList<>();
+    private String beforeMethod;
```

Comment:
Можно сразу сохранить `Method`
```suggestion
    private MethodbeforeMethod;
```

Аналогично и для `afterMethod`